src/shared/apis/api-hooks/alerts.ts

```diff
@@ -0,0 +1,29 @@
+import { GetData, PutData } from "./apiUtills";
+
+interface GetAlertsDataProps {
+  offset?: number;
+  limit?: number;
+}
+// 1. 유저의 알림 목록 조회 GET 요청
+export async function GetAlertsData({ offset, limit }: GetAlertsDataProps) {
+  const userId = sessionStorage.getItem("userId");
+  const queryParams = new URLSearchParams();
+  const parameters = { offset, limit };
+
+  Object.entries(parameters).forEach(([key, value]) => {
+    if (value !== undefined) {
+      queryParams.append(key, value.toString());
+    }
+  });
+
+  const url = `/users/${userId}/alerts?${queryParams.toString()}/`;
+  const { response, error, isLoading } = await GetData(url, true);
+  return { response, error, isLoading };
```

Comment:
일반적으로는 swr, react-query등에서 아래와 같은 인터페이스를 가지기 때문에 동일하게 가져가도 좋을 것 같습니다~
```suggestion
  return { data, error, isLoading };
```